fiftyone/core/labels.py

```
@@ -2103,3 +2188,547 @@ def _from_geo_json(d):
+
+        Returns:
+            the path to the exported `.fo3d` file
+
+        Raises:
+            ValueError: if no instances contain mesh geometry
+        """
+        import hashlib
+        import os
+
+        if not self.instances:
+            raise ValueError("Cannot export scene: no instance data")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)